packages/auth/src/grant/model.ts

```diff
@@ -17,10 +17,10 @@ export enum FinishMethod {
 }
 
 export enum GrantState {
-  Pending = 'pending',
-  Granted = 'granted',
-  Revoked = 'revoked',
-  Rejected = 'rejected'
+  Pending = 'PENDING',
+  Granted = 'GRANTED',
+  Revoked = 'REVOKED',
+  Rejected = 'REJECTED'
```

Comment:
I was thinking we should edit `20220504161932_create_grants_table` migration to change `state` to be an `enum` but there are more places that are guilty of this, so we can hold off for now